### PR TITLE
Add tiledb-py 0.1.0

### PR DESCRIPTION
**Pull request by jakebolewski**

### CODE REVIEW

File: recipes/tiledb-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: tiledb-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB-Py/archive/0.1.0.tar.gz
```
Comment (CJ-Wright) [inline recipes/tiledb-py/meta.yaml]: Please put the version number in the url as a jinja variable.

Comment (jakebolewski) [reply recipes/tiledb-py/meta.yaml]: Ok should be fixed

File: recipes/tiledb-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    TileDB  is an efficient multi-dimensional array management system which introduces 
+    a novel on-disk format that can store dense and sparse array data with 
+    support for fast reads and updates and reads. It features excellent compression, and an efficient 
+    parallel I/O system with high scalability.
+  doc_url: https://api-reference.tiledb.io/python-api.html
+  dev_url: https://github.com/TileDB-Inc/TileDB-Py
+
+extra:
+  recipe-maintainers:
+    - stavrospapadopoulos
```
Comment (CJ-Wright) [inline recipes/tiledb-py/meta.yaml]: @stavrospapadopoulos are you okay with being a maintainer?

File: recipes/tiledb-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - jakebolewski
+    - tdenniston
```
Comment (CJ-Wright) [inline recipes/tiledb-py/meta.yaml]: @tdenniston are you okay with being a maintainer?

File: recipes/tiledb-py/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install --tiledb=$PREFIX --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (CJ-Wright) [inline recipes/tiledb-py/meta.yaml]: Does this not also depend on `setuptools-scm`?

File: recipes/tiledb-py/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    - cython
+    - numpy 1.8.*  # [not (win and (py35 or py36))]
+    - tiledb 1.2.*
```
Comment (CJ-Wright) [inline recipes/tiledb-py/meta.yaml]: Is this a build requirement?

File: recipes/tiledb-py/meta.yaml (hunk 2 above)
Comment (stavrospapadopoulos) [reply recipes/tiledb-py/meta.yaml]: Yes

File: recipes/tiledb-py/meta.yaml (hunk 5 above)
Comment (jakebolewski) [reply recipes/tiledb-py/meta.yaml]: Yes

File: recipes/tiledb-py/meta.yaml (hunk 4 above)
Comment (jakebolewski) [reply recipes/tiledb-py/meta.yaml]: Not for this version, for later versions it will be a dependency.

File: recipes/tiledb-py/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB-Py/archive/{{ version }}.tar.gz
+  sha256: 8223e468f6c01bd82be4fdfee2a3b58af205b6ddd0f812fc4000b983304c8f66
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install --tiledb=$PREFIX --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/tiledb-py/meta.yaml]: Might put `$PREFIX` in double quotes like so `"$PREFIX"`. Just in case weird characters show up.

Comment (jakebolewski) [reply recipes/tiledb-py/meta.yaml]: Ok fixed, thanks for the suggestion.

File: recipes/tiledb-py/meta.yaml (hunk 3 above)
Comment (tdenniston) [reply recipes/tiledb-py/meta.yaml]: Yes.

**Comment by CJ-Wright** [comment]
@jakirkham @dougalsutherland are you okay with this recipe?

**Comment by dougalsutherland** [comment]
LGTM.

**Comment by jakebolewski** [comment]
Bump. Can this be merged?

**Comment by jakebolewski** [comment]
Thanks!

**Pull request merged**